src/molecules/Amount/Amount.native.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
```

Comment:
How above this API
```jsx
<Amount size="medium" currency="INR">
  1234
</Amount>;
```

The currency can be validated for the this https://www.currency-iso.org/dam/downloads/lists/list_one.xml -- for now we can just use INR but this opens up door for future support

The default value of currency can be set to INR as well

Reply:
I thought about it but supporting currency right now will add a lot more things like currency symbols based on what's selected, currency formatting based on selection. 
And since we're using a polyfill, we also need to ensure all the formatting is consistent across both the functions.

Not sure if it's worth the efforts as of now.

Reply:
I guess the intl is just a polyfill it should just work fine in the polyfill with the same options. See my below comment for the formatAmount function. It's just the matter of passing the currency to it nothing else